src/app/ReadClient.h

```diff
@@ -161,6 +161,12 @@ class ReadClient : public Messaging::ExchangeDelegate
          * @param[in] apReadClient The read client object of the terminated read or subscribe interaction.
          */
         virtual void OnDone(ReadClient * apReadClient) = 0;
+
+        /**
+         * OnDeallocatePaths will be called by AutoResubscribeClient, and deallocated the memory for mpEventPathParamsList and
```

Comment:
"This function is invoked when the ReadClient was configured to auto re-subscribe and the `ReadPrepareParams` was moved into this client for management. This will have to be free'ed appropriately by the application."

Reply:
updated, thanks